packages/data-context/src/data/coreDataShape.ts

```diff
@@ -197,6 +198,8 @@ export function makeCoreData (modeOptions: Partial<AllModeOptions> = {}): CoreDa
       chosenManualInstall: false,
       detectedBundler: null,
       detectedFramework: null,
+      // TODO: API to add third party frameworks to this list.
+      frameworks: CT_FRAMEWORKS.map((framework) => resolveComponentFrameworkDefinition(framework)),
```

Comment:
In the other part of this feature, https://github.com/cypress-io/cypress/issues/25680, we will add a function that doe something like:

```js
function addThirdPartyFramework (framework) {
  const resolvedFramework = resolveComponentFrameworkDefinition(framework)
  ctx.coreData.wizard.frameworks.push(resolvedFramework)
}
```

That will allow third parties to be here, too. They can easily have the correct shape by using the new `ComponentFrameworkDefinition` interface. That is defined in https://github.com/cypress-io/cypress/pull/25713/files#diff-dce5144c067a639808029e38cef9e4e90a64dec50aeec74e6064f3cb31a56981

Reply:
Example third party framework: https://github.com/cypress-io/cypress/pull/25713/files#diff-ba026001a8859c34bedee9aa69e0c18d19a305435efef49fa4e07cc0581e8d77R14-R41